inc/dynamic-css/block-editor-compatibility.php

```
@@ -197,34 +197,84 @@ function astra_load_modern_block_editor_ui( $dynamic_css ) {
 	$mobile_breakpoint = (string) astra_get_mobile_breakpoint();
 	/** @psalm-suppress InvalidCast */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
 
+	/** @psalm-suppress PossiblyUndefinedStringArrayOffset */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
+	$desktop_top_spacing = $blocks_spacings['desktop']['top'];
```

@premanshup / @imnavanath - Should we do a check if this exists instead of suppressing the error?

---

Done Resolved 👍